statsmodels/kernel_methods/tests/Multivariate.ipynb

```diff
@@ -0,0 +1,469 @@
+{
```

Comment:
why is this notebook in tests directory?
Also it contains a png file of a plot

notebooks shouldn't contain any output